build_helpers/buildStaticSite.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 set -e
-PATH=$(npm bin):$PATH
```

Comment:
I believe this the npm bin path was added so that the commands below (like `webpack`) can be executed.

So I initially planned on using `npx` or `npm exec` to execute them, but it turns out that these are no longer required.

As per the [docs](https://docs.npmjs.com/cli/v6/commands/npm-run-script) for `npm run`:
> In addition to the shell's pre-existing `PATH`, npm run adds `node_modules/.bin` to the `PATH` provided to scripts. Any binaries provided by locally-installed dependencies can be used without the `node_modules/.bin` prefix.